packages/rocketchat-ui/client/components/header/header.js

```diff
@@ -0,0 +1,124 @@
+/* globals fireGlobalEvent*/
```

Comment:
maybe we must rename this file to RoomHeader

Reply:
@ggazzo  But this header can be a section header too, like the ones in the preferences and settings screens